backend/src/test/java/codezap/member/repository/MemberJpaRepositoryTest.java

```diff
@@ -1,45 +1,139 @@
 package codezap.member.repository;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.jpa.repository.JpaRepository;
 
 import codezap.category.domain.Category;
 import codezap.category.repository.CategoryJpaRepository;
+import codezap.category.repository.CategoryRepository;
 import codezap.fixture.CategoryFixture;
+import codezap.fixture.MemberFixture;
+import codezap.global.exception.CodeZapException;
 import codezap.global.repository.JpaRepositoryTest;
 import codezap.member.domain.Member;
-import codezap.member.fixture.MemberFixture;
 import codezap.template.domain.Template;
 import codezap.template.repository.TemplateJpaRepository;
+import codezap.template.repository.TemplateRepository;
 
 @JpaRepositoryTest
 class MemberJpaRepositoryTest {
 
-    private final MemberJpaRepository memberJpaRepository;
-    private final TemplateJpaRepository templateJpaRepository;
-    private final CategoryJpaRepository categoryJpaRepository;
+    private final MemberRepository memberRepository;
+    private final TemplateRepository templateRepository;
+    private final CategoryRepository categoryRepository;
```

Comment:
테스트에서는 굳이 생성자를 선언할 필요가 없다고 생각하는데, 켬미는 어떻게 생각하나요?

final 키워드를 제거하고 필드에 `@Autowired`를 선언하면 라인 수를 줄일 수 있을 것 같아요~

```suggestion
    @Autowired
    private MemberRepository memberRepository;
    @Autowired
    private TemplateRepository templateRepository;
    @Autowired
    private CategoryRepository categoryRepository;
```